tools/scripts/rdl_post_process.py

```diff
@@ -35,6 +37,12 @@ def scrub_line_by_line(fname):
         has_reg_strb = re.search(r'\bdecoded_reg_strb\b', line)
         has_unpacked = re.search(r'\[\d+\]', line)
         has_struct = re.search(r'\bstruct\b\s*(?:unpacked)?', line)
+        is_endmodule = re.search(r'\bendmodule\b', line)
+        has_reset = re.search(r'\bnegedge.+\_b\b', line)
+        if (has_reset is not None and found_hard_reset is None):
+            substring = re.search(r"negedge (\w+.\w+)", line)
+            reset_name = substring.group(1)
+            found_hard_reset = re.search(r'hard|power',reset_name)
```

Comment:
This won't catch `cptra_pwrgood`

Reply:
wow, good catch - powergood isn't even a signal name we use.